doc/man1/flux-resource.rst

```diff
@@ -45,34 +45,34 @@ RESOURCE INVENTORY section below.
 COMMANDS
 ========
 
-**list** [-v] [-n] [-o FORMAT] [-s STATE,...]
-   Show scheduler view of resources.  One or more *-v,--verbose* options
-   increase output verbosity.  *-n,--no-header* suppresses header from output.
-   *-o,--format=FORMAT*, customizes output formatting (see below).
-   *-s,--states=STATE,...* limits output to specified resource states, where
-   valid states are "up", "down", "allocated", "free", and "all".  Note that
-   the scheduler represents "offline", "exclude", and "drain" resource states
-   as "down" due to its simplified interface with the resource service defined
-   by RFC 27.
+**list** [-n] [-o FORMAT] [-s STATE,...]
+   Show scheduler view of resources. The *-n,--no-header* option suppresses
+   header from output,  *-o,--format=FORMAT*, customizes output formatting
+   (see below), and  *-s,--states=STATE,...* limits output to specified
+   resource states, where valid states are "up", "down", "allocated",
+   "free", and "all".  Note that the scheduler represents "offline",
+   "exclude", and "drain" resource states as "down" due to its simplified
+   interface with the resource service defined by RFC 27.
 
 **info** [-s STATE,...]
    Show a brief, single line summary of scheduler view of resources.
    With *-s, --states=STATE,...*, limit the output to specified resource
    states as with ``flux resource list``. By default, the *STATE* reported
    by ``flux resource info`` is "all".
 
-**status**  [-v] [-n] [-o FORMAT] [-s STATE,...]
-   Show system view of resources.  One or more *-v,--verbose* options
-   increase output verbosity.  *-n,--no-header* suppresses header from output.
-   *-o,--format=FORMAT*, customizes output formatting (see below).
-   *-s,--states=STATE,...* limits output to specified resource states, where
-   valid states are "online", "offline", "avail", "exclude", "draining",
-   "drained", and "all". The special "drain" state is also supported, and
-   selects both draining and drained resources.
-
-**drain** [-f] [-u] [targets] [reason ...]
-   If specified without arguments, list drained nodes.  The *targets* argument
-   is an IDSET or HOSTLIST specifying nodes to drain.  Any remaining arguments
+**status**  [-n] [-o FORMAT] [-s STATE,...]
+   Show system view of resources.  The *-n,--no-header* suppresses header
+   from output, *-o,--format=FORMAT* customizes output formatting (see
+   below), and *-s,--states=STATE,...* limits output to specified resource
+   states, where valid states are "online", "offline", "avail", "exclude",
+   "draining", "drained", and "all". The special "drain" state is also
+   supported, and selects both draining and drained resources.
```

Comment:
need to add `--skip-empty`?

Reply:
Good catch! Fixed and pushed the result.